Assignment2/fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+	"@Context":{
+		"fullname": "http://www.upm/fullname/" ,
+		"hometown": "http://www.upm/hometown/" ,
+		"attributes": "http://www.upm/attributes/" ,
+ 		"age" : "attributes: age" ,
+		"almamater": "attributes: name" ,
+ 		"parents": "attributes: name" ,
+		"subjects": "http://www.upm/subjects/" ,
+		"teachers": "http://www.upm/teachers/" ,
+		"name" : "attributes: name" ,

```

Comment:
Check attribute definitions. You have extra blank spaces between prefixes and attribute names->e.g. "attributes: name" is wrong.